packages/nimble-components/src/theme-provider/index.ts

```diff
@@ -33,6 +38,11 @@ export const theme = DesignToken.create<Theme>({
  * @internal
  */
 export class ThemeProvider extends FoundationElement {
+    @attr({
```

Comment:
Could this be changed to `@attr` without the extra configuration? I'm guessing that you did this to be consistent with the other attributes in this file. Maybe we should update all of them unless there is a reason for the way they are written that I'm not aware of.